src/main/java/org/apache/commons/codec/digest/DigestUtils.java

```diff
@@ -99,6 +101,20 @@
         return updateDigest(messageDigest, data).digest();
     }
 
+    /**
+     * Reads through a RandomAccessFile using non-blocking-io (NIO) and returns the digest for the data
```

Comment:
First line of javadoc should end with a period.

Reply:
Fixed.